lib/src/validationsDispatcher.ts

```diff
@@ -53,22 +55,21 @@ export class ValidationDispatcher {
       this.isLastElement(index, numberOfItems);
   }
 
-  fireAllFieldsValidations(vm: any,
-    formFieldToViewModelKeyValues: Array<FormNameToFieldNameMapping>,
+  fireAllFieldsValidations(
+    vm: Object,
     validationFn: (vm, key, value) => Promise<FieldValidationResult>
   ): Array<Promise<FieldValidationResult>> {
 
-    let fieldValidationResultsPromises: Array<Promise<FieldValidationResult>> = [];
+    let fieldValidationResultsPromises: Promise<FieldValidationResult>[] = [];
 
-    if (this.areParametersDefined(vm, formFieldToViewModelKeyValues, validationFn)) {
-
-      formFieldToViewModelKeyValues.forEach((formFieldToViewModel: FormNameToFieldNameMapping) => {
-        const vmFieldValue = vm[formFieldToViewModel.vmFieldName];
+    if (this.areParametersDefined(vm, validationFn)) {
+      for (let vmKey in vm) {
```

Comment:
Other iteration could be using `Object.keys(vm).forEach(...)` but it's ok with that